### ISSUE TITLE
Mobile Text

### ISSUE THREAD

**Issue by DasBlueEyedDevil** [htck/bayeux#2]
Hi there!

I can't tell you how happy I was to find this, but I'm having a problem.  :-(

Everything seems to work except for adding text on a mobile device.  I click the background and I am presented the options to choose my font and color, but my keyboard doesn't get triggered, so I'm not able to physically enter text.

Any ideas?  :-)

**Comment by LeonardA-L** [comment]
Yes we're not adding an actual text field (that would trigger the mobile keyboard) but simply listening to key inputs. I'll take a look into manually triggering the event to enable the keyboard.
The app is overall not great for mobile, we worked on a responsive layout but that's pretty much it. I don't think we have plans for deeper development regarding mobile support though.

Thanks for your feedback :) I'll look into it

**Comment by eaugustine** [comment]
I'd like this feature as well. Even though you say it's not great for mobile, it's definitely workable except for the text. I'm able to add and manipulate images, and do pretty much everything else. Thanks for building this!

**Comment by Sosasees** [comment]
This app is definitely very optimized for Phones, so I can't see why they can't add a text field for phones.

This app is so close to being literally
Made for Phones, but then they stopped right before the finish line.
